Source/WTF/wtf/glib/GRefPtr.h

```diff
@@ -116,7 +140,10 @@ template <typename T> class GRefPtr {
     GRefPtr(HashTableDeletedValueType) : m_ptr(hashTableDeletedValue()) { }
     bool isHashTableDeletedValue() const { return m_ptr == hashTableDeletedValue(); }
 
-    T* get() const { return m_ptr; }
+    // Borrows the raw pointer from GRefPtr.
+    // The pointer is guaranteed to be valid for as long as GRefPtr holds an owning reference
+    // to that object.
+    T* /* transfer none */ get() const { return m_ptr; }
```

Comment:
```suggestion
    T* /* (transfer none) */ get() const { return m_ptr; }
```

Minor inconsistency.